app/models/mixins/supports_feature_mixin.rb

```diff
@@ -71,6 +71,7 @@ module SupportsFeatureMixin
     :resize                   => 'Resizing',
     :retire                   => 'Retirement',
     :smartstate_analysis      => 'Smartstate Analaysis',
+    :reboot_guest             => 'Reboot Guest Operation'
```

Comment:
could you add that in alphabetic order?
Also, as a side note, when adding something to the end of an array or hash, always have a trailing comma, so diffs only will show the changed lines - although thats debatable style :)